hc/api/models.py

```diff
@@ -89,7 +93,9 @@ def get_status(self):
         if self.last_ping + self.timeout + self.grace > now:
             return "up"
 
-        return "down"
+        if self.last_ping + self.timeout + self.grace < now:
+            return "down"
+        return "nag"
```

Comment:
When having two outcomes in an if statement, there is no need of writing the second if. Revert this back the previous code.